qcelemental/molparse/to_string.py

```diff
@@ -136,6 +138,26 @@ def to_dict(self) -> Dict:
 
         smol.extend(atoms)
 
+    if dtype == "orca":
```

Comment:
`elif` here will fix some tests.

note that if you `conda install pytest`, you can `pytest qcelemental/ -rws -v` in the main clone directory to test locally.

not mandatory, but it'd be nice to test the ghost atoms, too. there's a test you can mirror.